source/principles/data/index.html.md.erb

```diff
@@ -0,0 +1,187 @@
+---
+title: Data Principles
+last_reviewed_on: 2022-01-24
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+## Context
+
+Our work is governed by the [DfE Digital Enterprise Data Architecture Principles](https://dfe-digital.github.io/architecture/principles/enterprise-data-architecture-principles/); this document explains how we apply them to our Data Infrastructure. Please read the DfE principles before continuing, as this document implements, not replaces, them!
```

Comment:
Something needed here to be explicit on the context in which these apply (i.e. digital services). It's implicit from it being in the tech guidance, but might need calling out?

Reply:
Done